ntc_templates/templates/mikrotik_routeros_interface_bonding_print_detail.textfsm

```diff
@@ -0,0 +1,20 @@
+Value Required ID (\d+)
+Value STATUS (X|R)
+Value NAME (\S+)
+Value SLAVES (\S+(?:,)?)
+Value List DESCRIPTION (.+)
```

Comment:
```suggestion
Value List DESCRIPTION (.+?)
```

Reply:
Apologies, I had some local test files that I hadn't yet committed that cover that issue, but the CLI output is similar to when it comes to multiline comments [#1867](https://github.com/networktocode/ntc-templates/pull/1867).

I'm also going to add the \s* to remove leading whitespace too.